slack-moderator-words/events.go

```diff
@@ -115,7 +114,12 @@ func (h *handler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	if h.filters != nil {
 		for _, filter := range h.filters {
 			for _, word := range filter.Triggers {
-				if strings.Contains(event.Event.Text, word) {
+				matched, err := regexp.MatchString(word, event.Event.Text)
```

Comment:
TODO: amortize compiling the regexes on startup instead of once per regex one very event.